cli/src/commands/init.rs

```diff
@@ -90,9 +117,79 @@ fn prompt_group() -> io::Result<Option<String>> {
         return Ok(None);
     }
 
-    let group: String = Input::with_theme(&ColorfulTheme::default())
-        .with_prompt("Project Group (default: none)")
-        .interact_text()?;
+    let group: String =
+        Input::new().with_prompt("Project Group (default: none)").interact_text()?;
 
     Ok(Some(group))
 }
+
+/// Interactively ask for missing lockfile information.
+fn prompt_lockfile(
+    cli_lockfile: Option<&String>,
+    cli_lockfile_type: Option<&String>,
+) -> io::Result<(String, Option<String>)> {
+    if let Some(lockfile) = cli_lockfile {
+        return Ok((lockfile.clone(), cli_lockfile_type.cloned()));
+    }
```

Comment:
It seems like we should handle the case where `cli_lockfile_type` here is `None`. For example, if the user does this:

```
phylum init --lockfile yarn.lock foo
```

We should probably set the lockfile type to `yarn` because we can detect it by the filename.

It's the same behavior on any future `phylum analyze` because the command will look up the parser by the filename of the lockfile. But this way we can pre-calculate the parser and maybe even show a warning if it's not known (to discourage the use of the `auto` parser)